web_poet/page_inputs/stats.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+
+class Stats:
```

Comment:
I think we need an abc for this, because providers are not going to be providing this exact class. For example, scrapy-poet should provide another object, which writes to Scrapy stats.